build.xml

```
@@ -37,7 +37,7 @@
 		<exec executable="phpcs">
 			<arg value="--report=checkstyle" />
 			<arg value="--extensions=php" />
-			<arg value="--encoding=utf-8" />
+			<arg value="--encoding=UTF-8" />
```

```suggestion
			<arg value="--encoding=utf-8" />
```
I agree. Please revert this change or accept my cude change suggestions which will do the same.